ResourceManager/Xliff/TranslationUtil.cs

```diff
@@ -197,12 +197,7 @@ public static void TranslateItemsFromList(string category, ITranslation translat
 
         public static void TranslateProperty(string category, object obj, string propName, ITranslation translation)
         {
-            if (obj is null)
-            {
-                return;
-            }
-
-            var property = obj.GetType().GetProperty(propName, _propertyFlags);
+            var property = obj?.GetType().GetProperty(propName, _propertyFlags);
 
             if (property is null)
```

Comment:
It looks like a case.

Reply:
Do you mean a switch? I'm not sure what you mean exactly, sorry.

Reply:
Designated case: "Use null propagation"

Reply:
`property?.SetValue(obj, value, null);`

Reply:
Thanks, I see what you mean now.

Reply:
I found this option. But... 😄 

```
string value = translation.TranslateItem(category, propName, "Text", ProvideDefaultValue);

if (!string.IsNullOrEmpty(value))
{
    return;
}

var property = obj?.GetType()
    .GetProperty(propName, _propertyFlags);

if (property?.CanWrite is true)
{
    property.SetValue(obj, value, null);
}
```

Reply:
I'm not sure whether it's worth changing this. The current behaviour bails early when property is null, avoiding redundant work. Feel free to push to this branch if you'd like to see a change. For now, I can't see one that I like.